config/ftbquests/normal/chapters/08d906a0/0991db53.snbt

```diff
@@ -11,6 +11,14 @@
 		type: "item",
 		items: [{
 			item: "minecraft:dirt"
-		}]
+		}],
+		count: 8L,
```

Comment:
This quest required 8 dirt blocks, but starting island have 9 dirt blocks. That makes quest requirments too simple, not requiring any machine or even tool. Change count of required dirt to 10.